lib/features/popup-menu/PopupMenuComponent.js

```diff
@@ -54,6 +54,7 @@ export default function PopupMenuComponent(props) {
     scale,
     search,
     emptyPlaceholder,
+    searchFn,
```

Comment:
can we have some type definition for this?

Reply:
Also missing [here](https://github.com/bpmn-io/diagram-js/pull/932/files/688cc1179f6d1fcd295fef0678b81976ae292fe8#diff-20749ae7944fbc3d18a7ab06c462b53d0507502d6271eaa0e354394580b233c3R63), good catch.